pkg/machines/libvirt-dbus.js

```diff
@@ -359,14 +360,21 @@ LIBVIRT_DBUS_PROVIDER = {
                         call(connectionName, '/org/libvirt/QEMU', 'org.libvirt.Connect', 'StorageVolLookupByPath', [disk.source.file], TIMEOUT)
                                 .then(volPath => call(connectionName, volPath[0], 'org.libvirt.StorageVol', 'Delete', [0], TIMEOUT))
                     );
+                    storagePools.forEach(pool => {
```

Comment:
Maybe it's cleaner like :
const pool = storagePools.find(pool => pool.volumes.some(vol => vol.path === disk.source.file));
storageVolPromises.push(storagePoolRefresh(connectionName, pool.id));

In order to keep consistently with the other cases, but not really important :)